packages/langchain/src/orchestration/util.ts

```diff
@@ -123,6 +126,23 @@ function mapAzureOpenAiToLangchainToolCall(
   }
 }
 
+/**
```

Comment:
[req] Unrelated to this PR, but I see a function `mapAzureOpenAiToLangchainToolCall` in this file. Can we clean it up?